### ISSUE TITLE
Valid Wwise installation causes the build command to not generate o3de-install folder

### ISSUE THREAD

**Issue by LB-ArturSiewniak** [o3de/o3de#3837]
**Describe the bug**
It has been noticed that if a machine contains a valid Wwise installation under its default path (C:\Program Files (x86)\Audiokinetic\Wwise 2021.1.1.7601), then it will cause the build to throw errors related to Wwise (errors pasted below in the CMD Errors output section), and to not generate o3de-install folder and executable files under the <source>\o3de\build\windows_vs2019\bin\profile. This makes it effectively impossible to use the engine when following the default instructions listed on page https://docs.o3de.org/docs/welcome-guide/setup/setup-from-github/ under Build the engine > Pre-built SDK engine section.

It should be noted that following the Build the engine > Source engine section properly generates executables under the <source>\o3de\build\windows_vs2019\bin\profile, and completely removing Wwise from the machine along with the C:\Program Files (x86)\Audiokinetic folder allows the build listed under Build the engine > Pre-built SDK engine to properly generate executables and the o3de-install folder.

We also suspect that this issue could have been caused by this commit: https://github.com/o3de/o3de/commit/194205c4538161cafe06ba59b0323d462837798c.

**To Reproduce**
1. Setup a Wwise Launcher with the Wwise 2021.1.1.7601 installed.
2. Perform instructions listed on https://docs.o3de.org/docs/welcome-guide/setup/setup-from-github/ for the Pre-built SDK engine.

**Expected behavior**
The o3de-install is properly created and executables are generated.

**CMD Errors output**
```
Creating library C:/Builds/o3de/build/windows_vs2019/lib/profile/ImGui.lib and object C:/Builds/o3de/build/windows
  _vs2019/lib/profile/ImGui.exp
  unity_33_cxx.cxx
  ImGui.vcxproj -> C:\Builds\o3de\build\windows_vs2019\bin\profile\ImGui.dll
  unity_26_cxx.cxx
  unity_9_cxx.cxx
  unity_6_cxx.cxx
  unity_17_cxx.cxx
  unity_10_cxx.cxx
  unity_32_cxx.cxx
  unity_0_cxx.cxx
  unity_7_cxx.cxx
C:/Builds/o3de/Gems/AudioEngineWwise/Code/Source/Engine/Config_wwise.cpp(59,47): error C2039: 'ReadJsonFile': is not a
member of 'AzFramework::FileFunc' [C:\Builds\o3de\build\windows_vs2019\External\AudioEngineWwise-45aa7a62\Code\AudioEng
ineWwise.Static.vcxproj]
          auto outcome = AzFramework::FileFunc::ReadJsonFile(fileIoPath);
                                                ^
  C:\Builds\o3de\Code\Framework\AzFramework\AzFramework/FileFunc/FileFunc.h(28): note: see declaration of 'AzFramework:
  :FileFunc'
      namespace FileFunc
C:/Builds/o3de/Gems/AudioEngineWwise/Code/Source/Engine/Config_wwise.cpp(59,59): error C3861: 'ReadJsonFile': identifie
r not found [C:\Builds\o3de\build\windows_vs2019\External\AudioEngineWwise-45aa7a62\Code\AudioEngineWwise.Static.vcxpro
j]
          auto outcome = AzFramework::FileFunc::ReadJsonFile(fileIoPath);
                                                            ^
C:/Builds/o3de/Gems/AudioEngineWwise/Code/Source/Engine/Config_wwise.cpp(71,97): error C2661: 'AZ::JsonSerialization::L
oad': no overloaded function takes 2 arguments [C:\Builds\o3de\build\windows_vs2019\External\AudioEngineWwise-45aa7a62\
Code\AudioEngineWwise.Static.vcxproj]
          auto result = AZ::JsonSerialization::Load(*this, outcome.GetValue(), deserializeSettings);
                                                                                                  ^
C:/Builds/o3de/Gems/AudioEngineWwise/Code/Source/Engine/Config_wwise.cpp(72,12): error C3536: 'result': cannot be used
before it is initialized [C:\Builds\o3de\build\windows_vs2019\External\AudioEngineWwise-45aa7a62\Code\AudioEngineWwise.
Static.vcxproj]
          if (result.GetProcessing() != AZ::JsonSerializationResult::Processing::Completed)
             ^
C:/Builds/o3de/Gems/AudioEngineWwise/Code/Source/Engine/Config_wwise.cpp(95,47): error C2039: 'WriteJsonFile': is not a
 member of 'AzFramework::FileFunc' [C:\Builds\o3de\build\windows_vs2019\External\AudioEngineWwise-45aa7a62\Code\AudioEn
gineWwise.Static.vcxproj]
          auto outcome = AzFramework::FileFunc::WriteJsonFile(jsonDoc, filePath);
  unity_23_cxx.cxx
                                                ^
  C:\Builds\o3de\Code\Framework\AzFramework\AzFramework/FileFunc/FileFunc.h(28): note: see declaration of 'AzFramework:
  :FileFunc'
      namespace FileFunc
C:/Builds/o3de/Gems/AudioEngineWwise/Code/Source/Engine/Config_wwise.cpp(95,60): error C3861: 'WriteJsonFile': identifi
er not found [C:\Builds\o3de\build\windows_vs2019\External\AudioEngineWwise-45aa7a62\Code\AudioEngineWwise.Static.vcxpr
oj]
          auto outcome = AzFramework::FileFunc::WriteJsonFile(jsonDoc, filePath);
                                                             ^
  unity_18_cxx.cxx
  unity_1_cxx.cxx
  unity_2_cxx.cxx
```

**Desktop**
Device: PC
OS: Windows
Version 10
CPU AMD Ryzen 5 3600 6-Core Processor, 3600 MHz, 6 Core(s), 12 Logical Processor(s)
GPU NVIDIA GeForce RTX 2060 SUPER
Memory 16GB

**Comment by amznestebanpapp** [comment]
@amzn-phist do you know if there was there a fix for this recently?

**Comment by amzn-phist** [comment]
Yes, I fixed it here: https://github.com/o3de/o3de/pull/3989

**Comment by amznestebanpapp** [comment]
@LB-ArturSiewniak please let us know if there are still issues building. Thanks.

**Comment by LB-ArturSiewniak** [comment]
Issue verified as fixed on Development (d7e29c0) build.
